src/domain/ticketValidation/__tests__/TicketValidationPage.test.jsx

```diff
@@ -0,0 +1,138 @@
+      screen.getByText(
+        mockUseVerifyTicketQueryResponse.data.verifyTicket.eventName
+      )
+    ).toBeInTheDocument();
+  });
+
+  it('should display an invalid ticket message when validation fails', () => {
+    mockedUseVerifyTicketQuery.mockReturnValue({
+      ...mockUseVerifyTicketQueryResponse,
+      data: {
+        ...mockUseVerifyTicketQueryResponse.data,
+        verifyTicket: {
+          ...mockUseVerifyTicketQueryResponse.data.verifyTicket,
+          validity: false,
+        },
+      },
+    });
+    render(<TicketValidationPage />, { wrapper: ThemeProvider });
+    expect(screen.getByText('ticketValidation.invalid')).toBeInTheDocument();
+    expect(
+      screen.queryByText(
+        'ticketValidation.updateTicketAttended.switchButton.label'
+      )
+    ).not.toBeInTheDocument();
+  });
+
+  it('should display a success message when validation succeeds', () => {
+    mockedUseVerifyTicketQuery.mockReturnValue({
+      ...mockUseVerifyTicketQueryResponse,
+      data: {
+        ...mockUseVerifyTicketQueryResponse.data,
+        verifyTicket: {
+          ...mockUseVerifyTicketQueryResponse.data.verifyTicket,
+          validity: true,
+        },
+      },
+    });
+    render(<TicketValidationPage />, { wrapper: ThemeProvider });
+    expect(screen.getByText('ticketValidation.valid')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        'ticketValidation.updateTicketAttended.switchButton.label'
+      )
+    ).toBeInTheDocument();
+  });
+
+  it('should display an error message when error occurs', () => {
+    mockedUseVerifyTicketQuery.mockReturnValue({
+      ...mockUseVerifyTicketQueryResponse,
+      data: null,
+      error: { message: 'Error' },
+    });
+    render(<TicketValidationPage />, { wrapper: ThemeProvider });
+    expect(screen.getByText('ticketValidation.error')).toBeInTheDocument();
+  });
+
+  describe('form control', () => {
+    it("should display the 'Update Ticket Attended' switch button when the ticket is valid", () => {
+      mockedUseVerifyTicketQuery.mockReturnValue({
+        ...mockUseVerifyTicketQueryResponse,
+        data: {
+          ...mockUseVerifyTicketQueryResponse.data,
+          verifyTicket: {
+            ...mockUseVerifyTicketQueryResponse.data.verifyTicket,
+            validity: true,
+          },
+        },
+      });
+      render(<TicketValidationPage />, { wrapper: ThemeProvider });
+      expect(
+        screen.getByText(
+          'ticketValidation.updateTicketAttended.switchButton.label'
+        )
+      ).toBeInTheDocument();
+    });
+  });
+
+  describe('arrival status', () => {
+    it('should display the correct enrolment count', () => {
+      mockedUseVerifyTicketQuery.mockReturnValue(
+        mockUseVerifyTicketQueryResponse
+      );
+      render(<TicketValidationPage />, { wrapper: ThemeProvider });
+      expect(
+        screen.getByText('ticketValidation.arrivalStatus.label')
+      ).toBeInTheDocument();
+    });
```

Comment:
The test title talks about enrolment count, but test code does not mention enrolment count at all, so they are out-of-sync.